config/templates/terms-of-service.md

```diff
@@ -0,0 +1,99 @@
+## Introduction
+
+These terms of service (the "Terms") cover your access and use of Server Operator’s ("Administrator" we," or "us") instance, located at %{domain} (the "Instance").  These Terms apply solely to your use of the Instance as operated by the Administrator. Please note that we have no affiliation with Mastodon gGmbH (“Mastodon”) and these Terms do not contain any representations or warranties or other promises from Mastodon about your use of the Instance. If you would like to contact us for any reason, please direct all questions, comments, concerns and notices to us by following the instructions provided in the Notice section below.
```

Comment:
For ease of future-diff-review, readability, and to match the existing privacy policy markdown file ... hard wrap this whole file at 80 chars?